src/pages/shipping.js

```diff
@@ -22,290 +28,755 @@ export const shippingDashboard = (auth, route, goToSpecimenSearch) => {
     
+    if (Object.keys(detailedBoxes).length > 0) {
+        const sortedBoxKeys = Object.keys(detailedBoxes).sort();
+        sortedBoxKeys.forEach((box, i) => {
+            const currBox = detailedBoxes[box];
+            const bagKeys = Object.keys(currBox).filter(key => key !== 'boxData' && key !== 'undefined').sort((a, b) => a.split(/\s+/)[1] - b.split(/\s+/)[1]);
+            const boxStartedTimestamp = currBox.boxData[conceptIds.firstBagAddedToBoxTimestamp] ? formatTimestamp(Date.parse(currBox.boxData[conceptIds.firstBagAddedToBoxTimestamp])) : '';
+            const boxLastModifiedTimestamp = currBox.boxData[conceptIds.shippingShipDateModify] ? formatTimestamp(Date.parse(currBox.boxData[conceptIds.shippingShipDateModify])) : '';
+            const boxLocation = currBox.boxData[conceptIds.shippingLocation] ? locationConceptIDToLocationMap[currBox.boxData[conceptIds.shippingLocation]]["siteSpecificLocation"] : '';
+            const numTubesInBox = bagKeys.reduce((total, bagKey) => total + currBox[bagKey]['arrElements'].length, 0);
 
-    <!-- The Modal -->
-    <div class="modal fade" id="shippingModal" data-keyboard="false" data-backdrop="static" tabindex="-1" role="dialog" aria-hidden="true">
-        <div class="modal-dialog modal-dialog-centered" role="document">
-            <div class="modal-content sub-div-shadow">
-                <div class="modal-header" id="shippingModalHeader"></div>
-                <div class="modal-body" id="shippingModalBody">
-                </div>
-                <div class="modal-body"> 
-                    <h4 style="margin-bottom:0.8rem">Select Box or Create New Box</h4>
-                    <div id="create-box-success" class="alert alert-success" role="alert" style="display:none;">
-                      New box has been created
-                    </div>
-                    <div id="create-box-error" class="alert alert-danger" role="alert" style="display:none;">
-                      Please add a specimen or specimens to last box
-                    </div>
-                    <select class="selectpicker" id="shippingModalChooseBox" data-new-box="" style="font-size:1.4rem;"></select>
-                    <button type="button" class="btn btn-primary" id="modalAddBoxButton">Create New Box</button>
+            const currRow = table.insertRow(i + 1);
+            currRow.style['background-color'] = i % 2 === 1 ? 'lightgrey' : '';
+            
+            if (numTubesInBox > 0) {
+                currRow.innerHTML += renderBoxesToShipRow(boxStartedTimestamp, boxLastModifiedTimestamp, box, boxLocation, numTubesInBox);
+                const currBoxButton = currRow.cells[6].querySelector(".boxManifestButton");
+                currBoxButton.addEventListener("click", async () => {
+                generateBoxManifest(currBox);
+            });
+            }
+        });
+    }
+}
+
+export const populateTempNotification = () => {
+    let checkDate = false;
```

Comment:
Do we need `checkDate`? i don't understand the purpose of `checkDate`.

Reply:
Great catch. I had moved this code's location but had not inspected the functionality.

This functionality is handled by the element "tempMonitorChecked" and populateTempNotification() is a dead ref. Removed the function and reference.